.github/workflows/social-connect.yml

```diff
@@ -186,6 +182,12 @@ jobs:
     container:
       image: us-west1-docker.pkg.dev/devopsre/actions-runner-controller/celo-monorepo:node18
     needs: install-dependencies
+    if: |
+      contains(needs.install-dependencies.outputs.all_modified_files, 'packages') ||
+      contains(needs.install-dependencies.outputs.all_modified_files, '.github') ||
+      contains(needs.install-dependencies.outputs.all_modified_files, '.changeset') ||
+      contains(needs.install-dependencies.outputs.all_modified_files, 'packafe.json') ||
```

Comment:
typo `packafe.json` => `package.json`

Reply:
im interpreting this as "only run ci checks when a file matching one of these descriptions is changed" 

Im a bit worried about missing something. I think an explicit "dont run if only X file" is changed would feel safer. if not possible we should at least add 

* dockerfiles
* docs/examples

Reply:
Updated! I think we can run the general-test always.